File: CongressMemberService/Models/CongressMember.cs
using Common.Enums;

namespace CongressMemberService.Models
{
    public class CongressMember
    {
        public Guid Id { get; set; }

        public string? FirstName { get; set; }

        public string? MiddleName { get; set; }

        public string? LastName { get; set; }

        public string? Gender { get; set; }

        public string? Party {get; set; }

        public string? State { get; set; }

        public string? TwitterAccountName { get; set; }

        public Chamber Chamber { get; set; }
    }
}

File: sentiment_tweet_job/sentiment_tweet_job/models/congress_member.py
from django.db import models

class CongressMember(models.Model):
    id = models.CharField(primary_key=True)
    first_name = models.CharField()
    last_name = models.CharField()
    gender = models.CharField()
    party = models.CharField()
    state = models.CharField()
    twitter_account_name = models.CharField()
    chamber = models.IntegerField()
    middle_name = models.CharField(null=True)
